bootstrap/bootstrap.sh

```diff
@@ -28,8 +29,32 @@ sudo apt-get update
 sudo apt-get -y dist-upgrade
```

Comment:
No. It is not necessary. After bootstrap everything is fine. Except one thing: IPV4 and IPV6 will continue active and after reboot all connections will be using IPV4.